library.properties

```diff
@@ -1,5 +1,5 @@
 name=LoRa Serialization
-version=3.0.1
+version=3.0.2
```

Comment:
```suggestion
version=3.1.0
```
Given that you added features in #44, etc.?

Reply:
Good point. Changed as you proposed.